### PR TITLE
Contact form changes

### PR DESCRIPTION
**Pull request by mfadhili**
The pull request for the contact form updates in the portfolio project. 
The updates were done in a pair programming approach with @Shaheryar0054  and @mfadhili

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,9 @@
 ![Screenshot 2022-09-30 111842](https://user-images.githubusercontent.com/95047696/193204710-f62d4c4f-fec9-4177-9d99-f5ac6934ef68.png)
 ![Screenshot 2022-09-30 111926](https://user-images.githubusercontent.com/95047696/193204712-42591c94-d70e-4d62-a95e-f21b0ee5e7e0.png)
 ![Screenshot 2022-09-30 112019](https://user-images.githubusercontent.com/95047696/193204715-f39026ec-7890-4e77-a3b5-66e80638b2b3.png)
-
+![contact form](https://user-images.githubusercontent.com/95047696/194077434-0d53adf2-548e-4c5a-9aa5-05907ef1e488.png)
+# The message that I sent to Formspree
+![contact form message](https://user-images.githubusercontent.com/95047696/194077824-4ecb8ccb-83bf-4ca5-a724-237a452a2da2.png)
```
Comment (BertrandConxy) [inline README.md]: - [ ] This part is not needed in your README. I would suggest that you remove it 😃just to make it look professional
image

File: README.md [hunk 2]
```diff
@@ -16,7 +16,9 @@
 ![Screenshot 2022-09-30 111842](https://user-images.githubusercontent.com/95047696/193204710-f62d4c4f-fec9-4177-9d99-f5ac6934ef68.png)
 ![Screenshot 2022-09-30 111926](https://user-images.githubusercontent.com/95047696/193204712-42591c94-d70e-4d62-a95e-f21b0ee5e7e0.png)
 ![Screenshot 2022-09-30 112019](https://user-images.githubusercontent.com/95047696/193204715-f39026ec-7890-4e77-a3b5-66e80638b2b3.png)
-
+![contact form](https://user-images.githubusercontent.com/95047696/194077434-0d53adf2-548e-4c5a-9aa5-05907ef1e488.png)
```
Comment (BertrandConxy) [inline README.md:19]: - [ ] Please try to add more information in the `Getting started` section. This is where you talk about the procedures to clone and run your project locally. Important commands maybe included too 😃 
image

File: index.html [hunk 3]
```diff
@@ -237,6 +237,52 @@ <h4 class="heading-end">Skill</h4>
...
     </section>
+    <section class="form-section">
+        <div>
+            <h2 class="form-heading">I'm always <br> interested in hearing <br> about new projects, <br> so if you'd
+                like to <br> chat please get in <br> touch.</h2>
+        </div>
+        <div class="form-container">
+            <form action="https://formspree.io/f/xaykjlbb" id="form" method="POST">
+                <input type="text" class="input" maxlength="30" name="name" placeholder="Full name" required>
+                <input name="Email" class="input" type="email" placeholder="Email address">
```
Comment (BertrandConxy) [inline index.html]: - [ ] Please add the `required` attribute here on the email input as from the project requirements, all the fields are required 😉

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @mfadhili, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 

## Highlights 💪 
- No linter errors
- correct flow is used
- good PR

## Required Changes ♻️

- [ ] Good job so far 👏 First of all your README file needs some improvements. Starting with the project title. This is the Portfolio project not the hello Microverse project so please correct that. Also, the project description does not convey any information about the portfolio so please add a good project description. 👇 

image

- [ ] Again in your README. It is really great that you have added some screenshots to give visitors a quick preview of what your portfolio looks like. However, I can say that they are many. I think it would be better if you put one or two only. It looks more professional 😉

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — COMMENTED** [review]
Hi @mfadhili , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no linters running. Kindly fix that and resubmit your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (Abenezer-Tilahun) [reply README.md:19]: Fixed.

File: README.md (hunk 1 above)
Comment (Abenezer-Tilahun) [reply README.md]: Fixed.

File: index.html (hunk 3 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @mfadhili       , 

Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional  README
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**